crates/zizmor/Cargo.toml

```diff
@@ -63,6 +63,8 @@ tree-sitter.workspace = true
 tree-sitter-bash.workspace = true
 tree-sitter-powershell.workspace = true
 yamlpath.workspace = true
+diff = "0.1.13"
+jsonptr = "0.7"
```

Comment:
Nit: where does this dep get used? I tried to find it in `yaml_patch` but couldn't 🙂

Reply:
Leftover from previous testings. It is now manually implemented in the code.

Reply:
https://github.com/zizmorcore/zizmor/pull/858/commits/6bfb7b208210dd53dd34c6d4b5d390f44ded9db6